src/lib/apputils/net-server.c

```diff
@@ -1329,7 +1336,11 @@ process_tcp_connection_read(verto_ctx *ctx, verto_ev *ev)
     } else {
         /* msglen known. */
         socklen_t local_saddrlen = sizeof(struct sockaddr_storage);
-        struct sockaddr *local_saddrp = NULL;
+        krb5_address local_addr_buf = {0};
+        krb5_fulladdr local_addr = {
+            .address = &local_addr_buf,
```

Comment:
I will probably want to keep with the old pattern here of setting local_addr.address just before the call to init_addr(), rather than using a C99 initializer.

(I think maybe krb5_fulladdr should change so that the first field is krb5_address and not krb5_address *.  But that's out of scope for this PR.)